website/content/docs/agent/telemetry.mdx

```diff
@@ -269,6 +269,57 @@ resources will still work.
 This metric should be monitored to ensure that the license doesn't expire to prevent degradation of functionality.
 
 
+### Bolt DB Performance
+
```

Comment:
Would consider adding the Consul version when these metrics became available

Reply:
With versioned docs now I wasn't sure whether we still wanted to do that sort of thing? I guess it wouldn't hurt but I will reach out to some folks and figure out what our strategy should be for this type of thing now.

Reply:
I went ahead and added a little requirements section.